### DISCUSSION TITLE
Combobox - continuing with an error

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#1190] [Q&A] [answered]
Hello!

We're using an [auto-complete combobox](https://paste.twilio.design/components/combobox/#autocomplete-combobox) for a country list field in our new designs. 
Link to prototype [here](https://pqid1.csb.app/upgrade). 

During our usability tests, we noticed that users would type a country name wrong and continue to the next field without realising their mistake. Is there the possibility to add more error feedback to this. 

Below is a sample of one of the legacy comboboxes that capture what I'm trying to ask for. In the below case, the value in the dropdown is also cleared if entered incorrectly.
![image](https://user-images.githubusercontent.com/72031986/107808473-8ffe6000-6d61-11eb-9895-de7b52d65954.png)

**Comment by corinne-nagel** [comment, marked as answer]
Hey @PoojMenon , there is an error state for combobox here: https://paste.twilio.design/components/combobox#combobox-with-inline-error

Does that work for what you're thinking?

**Reply by PoojMenon** [reply]
I think that would work if we can do it on 'onBlur'. The value in the country field decides a few other things such as labels for state, zip code etc. Wanted to avoid them getting to the end of the form to realise country is wrong. 

Thanks Corinne!
